libs/cordova.force.js

```diff
@@ -1010,79 +1069,89 @@ cordova.define("com.salesforce.plugin.smartsync", function (require, exports, mo
     var SERVICE = "com.salesforce.smartsync";
 
     var exec = require("com.salesforce.util.exec").exec;
+    var defaultStoreName = "defaultStore";
```

Comment:
This has to match what's defined in `native`, correct? Can we remove this, so `native` is the single source of truth and we don't have to maintain them in both?